src/pages/integrations/bitpay-card/bitpay-settings/bitpay-settings.ts

```diff
@@ -58,6 +58,7 @@ export class BitPaySettingsPage {
       this.showAtHome
     );
     this.configProvider.set(opts);
+    this.bitPayCardProvider.setBitpayCardBannerStatus(this.showAtHome);
```

Comment:
Maybe we should keep `configProvider` to save the bannerStatus instead to add new key to storage.